gateway/src/main/java/io/camunda/zeebe/gateway/cmd/InvalidTenantRequestException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.gateway.cmd;
+
+public class InvalidTenantRequestException extends ClientException {
+
+  private static final String MESSAGE_FORMAT = "Invalid tenant-aware %s request: %s";
```

Comment:
❌ As this error message is directly responded to the client, the error messages must adhere to our [Error Guidelines](https://github.com/camunda/zeebe/wiki/Error-Guidelines)

Reply:
To add to this, I also wouldn't call it a tenant-aware request. I'd just say something along the lines of:

 - `Expected to process command X with tenant Y, but tenant contains illegal characters`
 - `Expected to process command X with tenant Y, but tenant is longer than 31 characters`
 -  `Expected to process command X with tenant Y, but multi-tenancy is disabled`